build/develop-env/lsstsqre/Dockerfile

```diff
@@ -31,46 +31,47 @@ COPY develop-env/lsstsqre/gitconfig ${HOME}/.gitconfig
 COPY develop-env/lsstsqre/setup_env.sh /home/saluser/.setup_salobj.sh
 
 RUN yum install -y https://repo.ius.io/ius-release-el7.rpm
-
+RUN RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | bash 
```

Comment:
Never seen two RUNs like this before. I'm sure this works. Does the outer run control the base after the pipe?